### DISCUSSION TITLE
New file location support

### DISCUSSION THREAD

**Discussion by brp-daniellofgren** [argenos/nldates-obsidian#104] [Q&A] [answered]
This feature request is simple.

Similarly as how the **core** Daily notes plugin allows you to designate a default **New file location** for files created with the plugin, I would like to see the same functionality added to `nldates`.

**Comment by argenos** [comment, marked as answer]
Nldates doesn't create any files, it simply adds links to your notes. You might want to check the Obsidian forum for a FR to create date-based notes in a folder.
